.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: ruby
 
 rvm:
-  - 2.0.0
-  - 2.1.6
-  - 2.2.3
+  - 2.4.5
+  - 2.5.3
 
 gemfile:
-  - gemfiles/ar40.gemfile
```

Comment:
I'd like to keep activerecord 4.2 compatibility as Guideline is still using that (upgrade to rails 5 is happening this quarter)

Reply:
@mnelson I've changed this back and added activerecord 4.x compatibility.

Reply:
Thanks. Looks like the test suite is very red at the moment.

Reply:
sorry for this, it's fixed now.